tests/resume.test.ts

```diff
@@ -220,3 +220,63 @@ describe('patch resume', () => {
     ).rejects.toThrowError(ForbiddenError);
   });
 });
+
+describe('delete resume', () => {
+  test('delete resume route successfully deletes user resume', async () => {
+    const conn = await DatabaseConnection.get();
+    const member = UserFactory.fake();
+    const resume = ResumeFactory.fake({ isResumeVisible: false, user: member });
+    await new PortalState()
+      .createUsers(member)
+      .createResumes(member, resume)
+      .write();
+
+    const params = { uuid: resume.uuid };
+
+    const storageService = Mocks.storage();
+    const resumeController = ControllerFactory.resume(
+      conn,
+      instance(storageService),
+    );
+    await resumeController.deleteResume(params, member);
+
+    const repository = conn.getRepository(ResumeModel);
+    const resumesStored = await repository.find({ user: member });
+
+    expect(resumesStored).toHaveLength(0);
+    verify(storageService.deleteAtUrl(resume.url)).called();
+  });
+
+  test('delete resume for another user throws a ForbiddenError', async () => {
+    const conn = await DatabaseConnection.get();
+    const [member, anotherMember] = UserFactory.create(2);
+    const resume = ResumeFactory.fake({ isResumeVisible: false, user: member });
+    await new PortalState()
+      .createUsers(member)
+      .createResumes(member, resume)
+      .write();
+
+    const params = { uuid: resume.uuid };
+    const resumeController = ControllerFactory.resume(conn);
+
+    await expect(
+      resumeController.deleteResume(params, anotherMember),
+    ).rejects.toThrowError(ForbiddenError);
```

Comment:
nit: let's assert the error message matches the exact error we'd expect to be thrown, instead of just the type. If we decide to throw a different kind of `ForbiddenError` in a future PR, then we would need to distinguish between the two errors in our tests, so we should do it right now instead to prevent future headaches.

Reply:
I found a method "toThrowErrorMatchingInlineSnapshot" that seems to be working. Would that create a new file since it compares snapshot?